internal/metamorph/store/postgresql/postgres_test.go

```diff
@@ -555,15 +558,114 @@ func TestPostgresDB(t *testing.T) {
 			MerklePath:      "",
 		}}
 
-		_, err := postgresDB.UpdateMined(ctx, txBlocks)
+		dataBeforeUpdate, err := postgresDB.Get(ctx, unminedHash[:])
+		require.NoError(t, err)
+
+		_, err = postgresDB.UpdateMined(ctx, txBlocks)
 		require.NoError(t, err)
 
 		dataReturned, err := postgresDB.Get(ctx, unminedHash[:])
 		require.NoError(t, err)
+		unmined.BlockHeight = 0
+		unmined.BlockHash = nil
+		unmined.StatusHistory = append(unmined.StatusHistory, &store.StoreStatus{
+			Status:    dataBeforeUpdate.Status,
+			Timestamp: dataBeforeUpdate.LastModified,
+		})
 		unmined.Status = metamorph_api.Status_MINED
+		unmined.LastModified = dataReturned.LastModified
```

Comment:
I think here we can set `unmined.LastModified = now` so that it is checked whether `dataReturned.LastModified == now`

Reply:
Replaced with `postgresDB.now()`